Project.toml

```diff
@@ -4,10 +4,12 @@ authors = ["Benedict Geihe <bgeihe@uni-koeln.de>", "Tristan Montoya <montoya.tri
 version = "0.1.0-DEV"
 
 [deps]
+HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
```

Comment:
Not sure if this is the best idea to keep two different compat settings for HDF5 unless there's a specific reason. Since this package is a "hard" downstream package of Trixi.jl, what about just `using Trixi.HDF5` instead of having it as an explicit dependency? Not sure if this has other downsides (@ranocha ?)

Reply:
If there is a breaking change in HDF5, it will be more difficult to resolve if we don't depend on it explicitly here.

Reply:
I was able to remove the dependency on HDF5 through my solution here https://github.com/trixi-framework/TrixiAtmo.jl/pull/51#discussion_r1861784766.

Reply:
Great!